### ISSUE TITLE
Multiple empty lines in JSDoc causes key error in React when using Info Addon.

### ISSUE THREAD

**Issue by strothj** [storybooks/storybook#3933]
### Bug summary

Creating a JSDoc comment for a prop type with multiple empty lines causes an error in the browser development console.

```
warning.js:33 Warning: Encountered two children with the same key, ``. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
    in td (created by Td)
    in Td (created by PropTable)
    in tr (created by PropTable)
    in tbody (created by PropTable)
    in table (created by Table)
    in Table (created by PropTable)
    in PropTable
    in Unknown (created by Story)
    in div (created by Story)
    in div (created by Story)
    in div (created by Story)
    in div (created by Story)
    in div (created by Story)
    in Story
```

### Steps to reproduce

1. Create a Story which utilizes the Info Addon:
2. Create a JSDoc comment for a prop type which contains more than one empty line:

Reproduction: https://github.com/strothj/storybook-info-addon-docgen-key-error

### Please specify which version of Storybook and optionally any affected addons that you're running

- @storybook/react 4.0.0-alpha.14
- @storybook/addon-something 4.0.0-alpha.14

### Affected platforms 

- _If UI related, please indicate browser, OS, and version_
- _If dependency related, please include relevant version numbers_
- _If developer tooling related, please include the platform information_

### Screenshots / Screencast / Code Snippets (Optional)

![image](https://user-images.githubusercontent.com/6731947/43410418-cb25ac90-93f4-11e8-8692-93eb5b4acf61.png)

1. Create a Story which utilizes the Info Addon:
```
import React from "react";
import { storiesOf } from "@storybook/react";
import SomeComponent from "../SomeComponent";

storiesOf("SomeComponent", module)
  .addParameters({ info: { inline: true } })
  .add("error", () => <SomeComponent />);
```

2. Create a JSDoc comment for a prop type which contains more than one empty line:
```
class SomeComponent extends React.Component {
  static propTypes = {
    /**
     * This is a required prop.
     *
     * With a multiple line description.
     *
     * After second line, causes error.
     */
    color: PropTypes.string
  };

  render() {
    return <div>Some Component</div>;
  }
}

export default SomeComponent;
```

**Comment by gombek** [comment]
@danielduan Hi, I created PR but some tests fails. Error logs looks like it is not because of my changes. Could you look on that and give me a hint?

**Comment by Hypnosphi** [comment]
Released as [`4.0.0-alpha.21`](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.21)
